data/provider/src/test/java/tech/pegasys/teku/api/blobselector/BlobSidecarSelectorFactoryTest.java

```diff
@@ -66,9 +67,9 @@ public void headSelector_shouldGetHeadBlobSidecars()
     when(client.getBlobSidecars(blockAndState.getSlotAndBlockRoot(), indices))
         .thenReturn(SafeFuture.completedFuture(blobSidecars));
 
-    final Optional<List<BlobSidecar>> result =
+    final Optional<BlobSidecarsAndMetaData> result =
         blobSidecarSelectorFactory.headSelector().getBlobSidecars(indices).get();
-    assertThat(result).hasValue(blobSidecars);
+    assertThat(Optional.of(result.get().getData())).hasValue(blobSidecars);
```

Comment:
NIT: I don't understand why we need to wrap it into an Optional for this assertion. Can't we do it like:
```
assertThat(result.get().getData()).isEqualTo(blobSidecars);
```

Reply:
This applies to other assertions in this class as well but I won't add a comment on each one :)